### PR TITLE
Let Neutron decide default network type

### PR DESCRIPTION
**Pull request by fnordahl**
If the user does not provide network type to the script let Neutron
decide instead of overriding to a hard coded value.

### CODE REVIEW

File: bin/neutron-ext-net-ksv3 [hunk 1]
```diff
@@ -92,19 +92,21 @@ if __name__ == '__main__':
                 'name': net_name,
                 'router:external': True,
                 'tenant_id': project_id,
-                'provider:network_type': opts.network_type,
             }
         }
 
         if opts.network_type == 'vxlan':
```
Comment (dshcherb) [inline bin/neutron-ext-net-ksv3:98]: Do we need an if block for `gre` as well?

File: bin/neutron-tenant-net-ksv3 [hunk 2]
```diff
@@ -90,16 +90,17 @@ if __name__ == '__main__':
                 'name': net_name,
                 'shared': opts.shared,
                 'tenant_id': project_id,
-                'provider:network_type': opts.network_type,
             }
         }
 
         if opts.network_type == 'vxlan':
```
Comment (dshcherb) [inline bin/neutron-tenant-net-ksv3:96]: Do we need an if block for `gre` as well?

File: bin/neutron-ext-net-ksv3 (hunk 1 above)
Comment (fnordahl) [reply bin/neutron-ext-net-ksv3:98]: Gah, yes, so much for the approach of doing as litte change as possible ;) Thx for catching it.

Updated, did not add handling specifcally for GRE, but instead added general handling of the something is specified vs. something is not specified case

File: bin/neutron-tenant-net-ksv3 (hunk 2 above)
Comment (fnordahl) [reply bin/neutron-tenant-net-ksv3:96]: Updated, did not add handling specifcally for GRE, but instead added general handling of the something is specified vs. something is not specified case

**Pull request merged**